swiftnav/dgnss_management.pyx

```diff
@@ -121,19 +154,11 @@ def dgnss_iar_get_single_hyp(num_dds):
   dgnss_management_c.dgnss_iar_get_single_hyp(&hyp[0])
   return hyp
 
-def get_dgnss_kf():
-  cdef nkf_t *kf = dgnss_management_c.get_dgnss_kf()
-  cdef KalmanFilter pykf = KalmanFilter()
-  memcpy(&(pykf.kf), kf, sizeof(amb_kf_c.nkf_t))
-  return pykf
-
-
-def get_stupid_state(num):
-  cdef np.ndarray[np.int32_t, ndim=1, mode="c"] N = \
-    np.empty(num, dtype=np.int32)
-  cdef s32 * c_N = dgnss_management_c.get_stupid_filter_ints()
-  memcpy(&N[0], c_N, num * sizeof(s32))
-  return N
+# def get_dgnss_kf():
+#   cdef nkf_t *kf = dgnss_management_c.get_dgnss_kf()
+#   cdef KalmanFilter pykf = KalmanFilter()
+#   memcpy(&(pykf.kf), kf, sizeof(amb_kf_c.nkf_t))
+#   return pykf
```

Comment:
+1, uncomment or remove entirely

its always in the git history if you need it again